docs/workspace.dsl

```diff
@@ -68,7 +68,7 @@ workspace "${SERVICE_NAME}" "Mozilla's service providing email and phone masks."
                     description "Collects metrics from other containers"
                     tags "Optional Application",in_other_managed_services
                     technology telegraf
-                    -> metrics_system "Sends metrics" "Telegraf"
+                    -> metrics_system "Pulls metrics" "Prometheus"
```

Comment:
comment (non-blocking): same comment here: does a Relay ENGR need to know that metrics are pulled vs. sent and that the underlying system is prometheus vs. telegraf? Or can that detail be in another doc and linked from our docs?